src/store/account/actions.js

```diff
@@ -114,6 +116,16 @@ export const logout = async function({ commit }) {
   }
 };
 
+
+export const getAccountData = async function({ commit, dispatch }) {
+  if (!this.state.account.accountName) {
+    return;
```

Comment:
in this case i think `commit("setData", undefined);` should be called

Reply:
done here [723fc65](https://github.com/telosnetwork/telos-wallet/pull/110/commits/723fc65b8e4935f053f3be2dbc2932e05db1aa00)